content/docs/operating/integrations.md

```diff
@@ -84,6 +84,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Rocket.Chat](https://rocket.chat/docs/administrator-guides/integrations/prometheus/)
   * [ServiceNow](https://github.com/FXinnovation/alertmanager-webhook-servicenow)
   * [Signal](https://github.com/dgl/alertmanager-webhook-signald)
+  * [SIGNL4 – Critical Mobile Alerting](https://www.signl4.com/blog/portfolio_item/prometheus-alertmanager-mobile-alert-notification-duty-schedule-escalation/)
```

Comment:
Just the basic name of the product is sufficient. This list is to help existing users of your product to find the integration.

In your example also the repeat_interval is less than the group_interval, which doesn't make sense. Resolve timeout should also be removed, that setting is the default and has no effect for Prometheus alerts - the setting does however cause confusion.

Reply:
Thanks and I did the changes as requested.

Reply:
The DCO check still fails. I am not sure why but I try to find out.